spec/controllers/articles_controller_spec.rb

```diff
@@ -122,7 +122,12 @@
         article1 = create(:article)
         article2 = create(:article)
         get :index
-        expect(assigns(:articles)).to eq([article2, article1])
+
+        assigned_articles = assigns(:articles)
+        assigned_articles_dates = []
```

Comment:
rather than creating a new array here you can reference the dates in assigned_articles like this : `assigned_articles[0]['created_at']`

Reply:
A much more eloquent solution! Thank you.